css/src/components/breadcrumbs.css

```diff
@@ -12,6 +12,22 @@
 
 .Breadcrumbs-link {
   margin: var(--spacing-m);
+  /* 160px */
+  max-width: calc(var(--spacing-2) * 10);
+  /* truncate */
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
Instead of writing different css properties please use `ellipsis--noWrap` className